app/components/newsletter/newsletter-header-overview.hbs

```diff
@@ -1,236 +1,239 @@
+                  @query={{hash showDraft=true}}
+                  @activeClass="active"
+                >
+                  <span class="auk-tabs__tab-inner">
+                    {{t "design"}}
+                  </span>
+                </LinkTo>
+              </li>
+              <li class="auk-tabs__tab">
+                <LinkTo
+                  @route="newsletter.print"
+                  @model={{await agenda.createdFor.id}}
+                  @query={{hash showDraft=false}}
+                  @activeClass="active"
+                >
+                  <span class="auk-tabs__tab-inner">
+                    {{t "definite"}}
+                  </span>
+                </LinkTo>
+              </li>
+              <li class="auk-tabs__tab">
+                <LinkTo
+                  data-test-newsletter-header-overview-nota-updates
+                  @route="newsletter.nota-updates"
+                  @activeClass="active"
+                >
+                  <span class="auk-tabs__tab-inner">
+                    {{t "newsletter-update-notas"}}
+                  </span>
+                </LinkTo>
+              </li>
+            {{/if}}
+          </ul>
+        </div>
       </div>
     </div>
-  </div>
-  <div class="auk-toolbar-complex__right">
-    <div class="auk-toolbar-complex__item">
+    <div class="auk-toolbar-complex__right">
       <div class="auk-toolbar-complex__item">
-        {{#if currentSession.isEditor}}
-        {{/if}}
-        <div class="auk-u-position-relative auk-u-inline-block">
-          <Auk::Button
-            @skin="secondary"
-            @icon={{if (not isShowingOptions) "chevron-down" "chevron-up"}}
-            {{action "showMultipleOptions"}}
-          >
-            {{t "actions"}}
-          </Auk::Button>
-          {{#attach-popover
-            renderInPlace=true
-            class="ember-attacher-popper vlc-hide-on-print"
-            hideOn="clickout click"
-            showOn="click"
-            animation="shift"
-            placement="bottom"
-          as |attacher|
-          }}
-            <ul class="vlc-dropdown-menu">
-              {{#if currentSession.isEditor}}
-                {{#if
-                  (or (is-pending agenda.createdFor.mailCampaign) isLoading)
-                }}
-                  <li class="vlc-dropdown-menu__item">
-                    <div class="auk-loader-wrapper auk-loader-wrapper--small auk-u-p-2">
-                      <div
-                        class="auk-loader"
-                        role="alert"
-                        aria-busy="true"
-                      ></div>
-                      {{t "newsletter-loading-text"}}
-                    </div>
-                  </li>
-                {{else if (await agenda.createdFor.mailCampaign)}}
-                  {{#if (not (await agenda.createdFor.mailCampaign.isSent))}}
+        <div class="auk-toolbar-complex__item">
+          {{#if currentSession.isEditor}}
+          {{/if}}
```

Comment:
deze if stond er al, maar viel me gewoon op. vermoedelijk willen we gewoon alle opties hierin steken.
Ik zal deze aanpassen in een latere commit/branch om conflicten te vermijden